html/changelogs/geeves-awaysite_pin_fix.yml

```diff
@@ -0,0 +1,6 @@
+author: Geeves
+
+delete-after: True
+
+changes: 
+  - bugfix: "Away site pins should only allow the gun to fire at away sites now."
```

Comment:
`[...]should[...]`
Did you test that bugfix ?
I think there are two issues with it:
- gun.z might not work for the same reasons as src.z
- It would imho be better to fix this by adding a list of away mission z levels (which is automatically generated when the away mission level(s) are loaded in.